sdk/dotnet/Dagger.SDK.Tests/GraphQL/QueryBuilderTest.cs

```diff
@@ -0,0 +1,83 @@
+using Dagger.SDK.GraphQL;
+
+namespace Dagger.SDK.Tests.GraphQL;
+
+public class QueryBuilderTest
+{
+    [Fact]
```

Comment:
I know the original was using NUnit, and I've been using XUnit for years. Thoughts on `https://devblogs.microsoft.com/dotnet/introducing-mstest-sdk/` ?

Reply:
@pjmagee sorry for late reply. I think it would be great to migrate to it. I add an issue on dagger-dotnet-sdk about this https://github.com/wingyplus/dagger-dotnet-sdk/issues/9